---
http_interactions:
- request:
    method: post
    uri: https://cognito-idp.us-east-1.amazonaws.com/
    body:
      encoding: UTF-8
      string: '{"ClientId":"76jcq337aei27va9q9gtaep0dh","Username":"nick.watson@claritybizsol.com","Password":"changeME123!","UserAttributes":[{"Name":"email","Value":"nick.watson@claritybizsol.com"}]}'
    headers:
      Content-Type:
      - application/x-amz-json-1.1
      Accept-Encoding:
      - ''
      User-Agent:
      - aws-sdk-ruby3/3.38.0 ruby/2.5.3 x86_64-darwin18 aws-sdk-cognitoidentityprovider/1.10.0
      X-Amz-Target:
      - AWSCognitoIdentityProviderService.SignUp
      Content-Length:
      - '186'
      Accept:
      - "*/*"
  response:
    status:
      code: 400
      message: Bad Request
    headers:
      Date:
      - Fri, 16 Nov 2018 13:34:40 GMT
      Content-Type:
      - application/x-amz-json-1.1
      Content-Length:
      - '68'
      Connection:
      - keep-alive
      X-Amzn-Requestid:
      - 5e5531be-e9a4-11e8-ab3e-bb4a5df5ad66
      X-Amzn-Errortype:
      - 'UsernameExistsException:'
      X-Amzn-Errormessage:
      - User already exists
    body:
      encoding: UTF-8
      string: '{"__type":"UsernameExistsException","message":"User already exists"}'
    http_version: 
  recorded_at: Fri, 16 Nov 2018 13:34:40 GMT
recorded_with: VCR 4.0.0
